Make Game::BuildProject public to fix "error: 'BuildProject' is a private
member of 'Game'".
https://savannah.nongnu.org/bugs/?47070
https://bitbucket.org/giszmo/glob2/commits/c4da01699846179d8bf21e8dae2b973158ec0775
--- src/Game.h.orig	2009-08-29 15:39:06.000000000 -0500
+++ src/Game.h	2018-10-17 20:39:14.000000000 -0500
@@ -149,6 +149,7 @@
 		BOTTOM_TO_TOP
 	};
 	
+public:
 	struct BuildProject
 	{
 		int posX;
@@ -159,6 +160,7 @@
 		int unitWorkingFuture;
 	};
 	
+private:
 	///Initiates Game
 	void init(GameGUI *gui, MapEdit* edit);
 
